docs/MIGRATE.md

```diff
@@ -0,0 +1,21 @@
+# Migrating from `janimo/axolotl`
+
+1. Download and install the app from the OpenStore; do not launch the app!
+2. Copy the directory `/home/phablet/.local/share/textsecure.jani/.storage` to
+   `/home/phablet/.local/share/textsecure.nanuc/.storage`
+3. Copy the file `/home/phablet/.config/textsecure.jani/config.yml` to
+   `/home/phablet/.config/textsecure.nanuc/config.yml`.
+   Edit the copied file by changing `storageDir: /home/phablet/.local/share/textsecure.nanuc/.storage` (not strictly required: also
+   update `userAgent: TextSecure 0.3.18 for Ubuntu Phone` to reflect the current version).
```

Comment:
@nanu-c I guess this 0.3.18 isn't the good value anymore?

Reply:
I have to check if the useragent is used at all from that file, because it's generated magically from the current version, but let's see